Sources/Fuzzilli/Mutators/OperationMutator.swift

```diff
@@ -26,6 +26,22 @@ public class OperationMutator: BaseInstructionMutator {
         var newOp: Operation
         
         b.trace("Mutating next operation")
+
+        func replaceRandomElement<T>(in set: inout Set<T>, generatingRandomValuesWith generator: () -> T) {
+            let removedElem: T? = nil
+            if let removedElem = set.randomElement() {
```

Comment:
Is this really fine? (instead of the `let removedElem = set.randomElement() else { return }`)? When you change the size of the set, don't you then also need to change the inputs of the instruction? I think it's easier to just bail and not mutate an empty set at all (we should anyway never see that in practice)

However, that also means that the loop in lines 36+ must succeed, or else the new set will be smaller... Maybe replace the `break` in line 41 with a `return`, then put

```swift
// Failed to insert a new element, so just insert the removed element again as we must not change the size of the set
set.insert(removedElem)
```
after the loop?